test/markdown.js

```diff
@@ -23,7 +23,7 @@ describe('Markdown', function() {
       expect(this.$).have.selector('p.sg')
     })
     it('html template', function() {
-      expect(this.html).match(/doctype html/)
+      expect(this.html).match(/!DOCTYPE html/)
```

Comment:
doctype is case insensitive. https://html.spec.whatwg.org/#the-doctype

Reply:
Yes but prettier puts it in capital letters

Reply:
Honestly, I've just fixed the tests without thinking. The project needs a refacto but I don't have the time to do it